nvflare/private/fed/server/server_command_agent.py

```diff
@@ -73,7 +73,7 @@ def execute_command(self, request: CellMessage) -> CellMessage:
                 if not client:
                     return make_reply(
                         ReturnCode.AUTHENTICATION_ERROR,
-                        "Request from invalid client: missing client token",
+                        "Request from client: missing client token",
```

Comment:
"Request missing client login token or has an invalid token"

Reply:
Looks like it could be either case: the token is missing, or client couldn't found from the token.
Could be changed to "missing or invalid token".